sunpy/coordinates/ephemeris.py

```diff
@@ -109,15 +113,110 @@ def get_earth(time='now'):
     return earth
 
 
-def get_sun_B0(time='now'):
+@add_common_docstring(**_variables_for_parse_time_docstring())
+def get_horizons_coord(body, time='now', id_type='majorbody'):
+    """
+    Queries JPL HORIZONS and returns a `~astropy.coordinates.SkyCoord` for the location of a
+    solar-system body at a specified time.  This location is the instantaneous or "true" location,
+    and is not corrected for light travel time or observer motion.
+
+    .. note::
+        This function requires the Astroquery package to be installed and
+        requires an Internet connection.
+
+    Parameters
+    ----------
+    body : `str`
```

Comment:
Is there a list of strings somewhere that identify the bodies?  Might be good to point to that.

Reply:
Unfortunately, nothing that can be easily referenced (that I have found).  JPL HORIZONS's web form to search through their database can't be linked to directly.  Honestly, just asking for what you want is typically the best way to find out if it's there or not.

Reply:
Okay, I take that back.  It looks like I can link reasonably to their web form, so that can be added to the docstring.